docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -33,7 +33,7 @@ When you're using Visual Studio 2017, many of those steps are handled for you, w
 
 However, just because Visual Studio makes those steps automatic doesn't mean that you don't need to know what's going on underneath with Docker. Therefore, the following guidance details every step.
 
-![1 - Code your app](./media/image2.png)
+![Image of the Step 1 tile from the step-by-step workflow.](./media/docker-app-development-workflow/step-1-code-your-app.png)
```

Comment:
Don't really think the alt-text here has much value, would just leave it mostly as it was.

```suggestion
![Image for Step 1.](./media/docker-app-development-workflow/step-1-code-your-app.png)
```

Also the title right below says it all, so no need to add anything (I missed that originally)